api/config.py

```diff
@@ -61,7 +66,20 @@
 
 def apply_env_variables(config):
     # Overwrite default config values with SCITRAN env variables if available
+
+    # Load auth config from file if available
+    if 'SCITRAN_AUTH_CONFIG_FILE' in os.environ:
+        auth_config = config['auth']
+        file_path = os.environ['SCITRAN_AUTH_CONFIG_FILE']
+        with open(file_path) as config_file:
```

Comment:
Relevant to the PR. How we load in environment variable config options now works because they are all known key-values we can iterate over. Adding dynamic keys ('google', 'ldap', etc), we can't use the same logic. Using a file to load the config at startup time is the easiest way to support that.